From 5dc264e4b3331626447fa556f40ba563e7128399 Mon Sep 17 00:00:00 2001
From: Rahul Yadav <rahuly.simform@gmail.com>
Date: Wed, 11 Jan 2023 15:06:59 +0530
Subject: [PATCH] gfgf

---
 AppDelegate.swift | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/AppDelegate.swift b/AppDelegate.swift
index 6d0eac9..860a922 100644
--- a/AppDelegate.swift
+++ b/AppDelegate.swift
@@ -4,7 +4,10 @@
 //
 //  Created by Rahul Yadav on 09/01/23.
 //
+<<<<<<< HEAD
 //New Comment added
+=======
+>>>>>>> 376f325 (New File Added)
 
 import UIKit
 
-- 
2.37.0

